openseadragon-canvas-overlay.js

```diff
@@ -2,8 +2,8 @@
 
 (function() {
     
-    $ = window.OpenSeadragon;
-    
+    var $ = window.OpenSeadragon;
```

Comment:
Without this `var` this plugin was overriding jQuery if you had it on the page.

Reply:
thanks!